package/base-files/files/lib/preinit/10_indicate_preinit

```diff
@@ -63,6 +63,24 @@ preinit_config_switch() {
 	json_select ..
 }
 
+preinit_config_port() {
```

Comment:
`preinit_config_port`, or `preinit_config_netdev_name`?

Reply:
I think this naming is fine.